### DISCUSSION TITLE
关于在win10桌面上的问题

### DISCUSSION THREAD

**Discussion by Galen-87** [shuax/MouseInc.Settings#131] [Q&A]
在win10桌面的情况下,通用手势里面的删除,复制,粘贴对文件没有效果,不知道是怎么回事,还有能不能新增一个设置完成之后一键导入/导出设置的选项,不然如果换台电脑又要重新设置了.比较麻烦
还有,我想请问一下,能否给手势设置一个针对某个程序排除的选项,用浏览器的的手势有的手势比较好用

**Comment by shuax** [comment]
复制mouseinc.json就是配置。画Z可以排除程序。
